spec/support/shared_examples/page_with_top_nav.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.shared_examples 'page with top nav' do
+  it 'should have top nav link to home' do
+    render
+    expect(rendered).to have_selector("#main-menu a", text: 'Home')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.